### ISSUE TITLE
Move Location Visit Section to Top

### ISSUE THREAD

**Issue by mripp** [NWFWMD-IT/Hydrologic-Monitoring-Project#146]
Move Location Visit section to top of survey, directly after FLUWID and site photos

**Comment by mripp** [comment]
Discussion is needed on implications on moving this up. It is difficult to show the Data Logger, Sensor information outside of the repeat. Can show the option to display in a table like format

**Comment by mripp** [comment]
On 2/12 meeting we discussed that the Visit Date and Staff need to be shown directly after the FLUWID area. 
This format is ideal
![Image](https://github.com/NWFWMD-IT/Hydrologic-Monitoring-Project/assets/55856222/0ffc61d6-b2d6-49be-b37c-e07167f5fdca)

Otherwise I discussed the option to have a 'Visit Date' and 'Staff' question formatted as NULL in the top of the survey. And this would calculate down into the actual fields. This could work but it doesn't fix the option to have MP moved down in the survey #145
